app/models/trip.rb

```diff
@@ -0,0 +1,20 @@
+class Trip < ApplicationRecord
+  belongs_to :passenger
+  belongs_to :driver
+  
+  validates :date, presence: true
+  validates :driver, presence: true
+  validates :passenger, presence: true
+  
+  def self.all_in_alpha_order
+    return Trip.all.order(date: :asc)
+  end
+  
+  def convert_to_dollars(pennies)
+    if pennies.nil?
+      return 0
+    else
+      (pennies / 100.0).round(2)
```

Comment:
Love that y'all made a separate method for this! That said, the fact that this is repeated in three different classes is a sign that it probably belongs in its own class. It could be put in a class (maybe a new class called Money, for instance) and made a class method so that an instance of money isn't needed to call it.